HW_1/task_4.py

```diff
@@ -0,0 +1,15 @@
+# Задача 4
+# Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, 
+# если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# Пример:
+# 6 -> 1 4 1
+# 24 -> 4 16 4
+# 60 -> 10 40 10
+
+S = (int(input('Введите количество журавликов: ')))
+if S % 6 == 0:
+    print(f'Катя сделала {int(S * 2 / 3)} шт. ')
+    print(f'Петя сделал {int(S * 1 / 6)} шт. ')
+    print(f'Серёжа сделал {int(S * 1 / 6)} шт. ')
```

Comment:
Во-первых, функцию `int` можно не использовать, вместо этого можно использовать целочисленное деление. Во-вторых, зачем умножать на 1?
```suggestion
    print(f'Катя сделала {S * 2 // 3} шт. ')
    print(f'Петя сделал {S // 6} шт. ')
    print(f'Серёжа сделал {S // 6} шт. ')
```